sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/main/resources/application-mi.yaml

```diff
@@ -17,6 +17,8 @@ spring:
         checkpoint-storage-account: [checkpoint-storage-account]
         checkpoint-container: [checkpoint-container]
     stream:
+      function:
```

Comment:
why move this here?

Reply:
Hope this is clearer, I put it at the bottom level of spring.stream.

Reply:
okay, even though it makes no difference to me.